www/index.html

```diff
@@ -906,6 +917,41 @@
           </div>
         {{/each}}
       </script>
+
+      <script type="text/template7" id="galleryTemplate">
+        {{#each album}}
+          <a href="album/{{id}}">
+            <div class="albums">
+              <div class="album-full-background" style="background-image: url({{thumb}})"></div>
+              <div class="album-text">
+                <div class="title"><p> {{title}} </p></div>
+                <div><i class="fa fa-calendar"></i> {{js "new Date(this.start_date).shortDateString()"}}</div>
+                <div><i class="fa fa-map-marker"></i> {{location}}</div>
+                <div><i class="fa fa-image"></i> {{image_count}}</div>
+              </div>
+            </div>
+          </a>
+        {{/each almbum}}
+      </script>
+
+      <script type="text/template7" id="albumTemplate">
+        <div class="block-title">{{title}}</div>
+        <div class="album-info-block">
```

Comment:
You can remove `album-info-block` and just use F7:s `block` class instead. Don't forget to edit the css file as well.